### PR TITLE
feat: Introduce - adds controller BDD tests

### PR DESCRIPTION
**Pull request by soluchok**
* auto-execute `outofband` events
* adds introduce  `skip_proposal` test

Part of #1880

Signed-off-by: Andrii Soluk <isoluchok@gmail.com>

### CODE REVIEW

File: pkg/controller/command/outofband/command.go [hunk 1]
```diff
@@ -73,7 +73,11 @@ func New(ctx outofband.Provider) (*Command, error) {
 	// this code listens for the action events and does nothing
 	// this trick is used to avoid error such as `no clients are registered to handle the message`
 	go func() {
-		for range actions {
+		for action := range actions {
+			// TODO currently we do not have the possibility to continue with the protocol through the REST API
+			//  The client must provide a method which will return all pending actions.
+			//  Also, the client must provide a method that will allow us to continue with the protocol by action ID.
```
Comment (llorllale) [inline pkg/controller/command/outofband/command.go:79]: Also needs to provide a mechanism for the client to specify the embedded request message.

File: test/bdd/fixtures/agent-rest/docker-compose.yml [hunk 2]
```diff
@@ -16,6 +16,7 @@ services:
       - ARIESD_INBOUND_HOST_EXTERNAL=${HTTP_SCHEME}@http://alice.aries.example.com:${ALICE_INBOUND_PORT}
       - ARIESD_WEBHOOK_URL=http://${ALICE_WEBHOOK_CONTAINER_NAME}:${ALICE_WEBHOOK_PORT}
       - ARIESD_DEFAULT_LABEL=alice-agent
+      - ARIESD_LOG_LEVEL=DEBUG
```
Comment (llorllale) [inline test/bdd/fixtures/agent-rest/docker-compose.yml:19]: please revert logging level

File: test/bdd/pkg/didexchange/didexchange_controller_steps.go [hunk 3]
```diff
@@ -101,8 +101,14 @@ func (a *ControllerSteps) RegisterSteps(s *godog.Suite) { //nolint dupl
 }
 
 // EstablishConnection establishes connection between two agents through did exchange protocol
-func (a *ControllerSteps) EstablishConnection(inviter, invitee string) error {
-	return a.performDIDExchange(inviter, invitee)
+func (a *ControllerSteps) EstablishConnection(inviters, invitee string) error {
```
Comment (llorllale) [inline test/bdd/pkg/didexchange/didexchange_controller_steps.go:104]: Let's phase out did-exchange and instead add this in this out-of-band bdd package

Comment (soluchok) [reply test/bdd/pkg/didexchange/didexchange_controller_steps.go:104]: We do not have `outofband_controller_steps` tests for now. https://github.com/hyperledger/aries-framework-go/issues/1891

**Pull request merged**